<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,0,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="x_label">
              <property name="text">
               <string>X coordinate coil</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="x_box">
              <property name="suffix">
               <string/>
              </property>
              <property name="minimum">
               <double>-999.990000000000009</double>
              </property>
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="y_label">
              <property name="text">
               <string>Y coordinate coil</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="y_box">
              <property name="minimum">
               <double>-999.990000000000009</double>
              </property>
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="z_label">
              <property name="text">
               <string>Z height for test</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="z_box">
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="z_label_2">
              <property name="text">
               <string>Scanning range</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="range_box"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="z_label_3">
              <property name="text">
               <string>Speed</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QDoubleSpinBox" name="speed_box"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="temp_label">
              <property name="text">
               <string>Nozzle temperature</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QSpinBox" name="temp_box">
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="temp_label_2">
              <property name="text">
               <string>Bed temperature</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QSpinBox" name="bed_temp_box">
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>mm</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>mm</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>mm</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>mm</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>mm/s</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>°C</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>°C</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="duet_label">
            <property name="text">
             <string>Duet port</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="duet_combo">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ldc_label">
            <property name="text">
             <string>LDC1101EVM port</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ldc_combo">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="connect_button">
              <property name="text">
               <string>Connect</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="reload_button">
              <property name="text">
               <string>Reload</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Tools to calibrate</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="tool_list_list">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::MultiSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectItems</enum>
              </property>
              <item>
               <property name="text">
                <string>0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>7</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>9</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Reference tool</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="ref_combo">
              <item>
               <property name="text">
                <string>0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>7</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>9</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="ascend_box">
              <property name="text">
               <string>ascend</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="tristate">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="descend_box">
              <property name="text">
               <string>descend</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="fan_box">
              <property name="text">
               <string>Fan</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="homing_box">
              <property name="text">
               <string>Homing</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,0,0,0,0,0">
        <item>
         <widget class="PlotWidget" name="sig_graph" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="clear_figure_button">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>clear figure</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stop_button">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>stop</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="test_sensor_button">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Test sensor</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="cal_y_button">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Calibrate Y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cal_x_button">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Calibrate X</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Rounds:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="rounds_y_spinner">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Rounds:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="rounds_x_spinner">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QPushButton" name="apply_offsets_button">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Apply offsets</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="update_coil_button">
            <property name="minimumSize">
             <size>
              <width>115</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Update coil position</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Datafile name:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="filename_line">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>data.mat</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="output_label">
      <property name="text">
       <string>output:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="output_terminal">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
